sdk/textanalytics/Azure.AI.TextAnalytics/tests/RecognizePiiEntitiesTests.cs

```diff
@@ -0,0 +1,166 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using NUnit.Framework;
+
+namespace Azure.AI.TextAnalytics.Tests
+{
+    public class RecognizePiiEntitiesTests : TextAnalyticsClientLiveTestBase
+    {
+        public RecognizePiiEntitiesTests(bool isAsync) : base(isAsync) { }
+
+        private const string singleEnglish = "A developer with SSN 859-98-0987 whose phone number is 800-102-1100 is building tools with our APIs.";
+
+        private static List<string> batchConvenienceDocuments = new List<string>
+        {
+            "A developer with SSN 859-98-0987 whose phone number is 800-102-1100 is building tools with our APIs.",
+            "Your ABA number - 111000025 - is the first 9 digits in the lower left hand corner of your personal check."
+        };
+
+        private static List<TextDocumentInput> batchDocuments = new List<TextDocumentInput>
+        {
+            new TextDocumentInput("1", "A developer with SSN 859-98-0987 whose phone number is 800-102-1100 is building tools with our APIs.")
+            {
+                 Language = "en",
+            },
+            new TextDocumentInput("2", "Your ABA number - 111000025 - is the first 9 digits in the lower left hand corner of your personal check.")
+            {
+                 Language = "en",
+            }
+        };
+
+        [Test]
+        public async Task RecognizePiiEntitiesTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            string document = singleEnglish;
+
+            PiiEntityCollection entities = await client.RecognizePiiEntitiesAsync(document);
+
+            Assert.AreEqual(2, entities.Count);
+
+            var entitiesList = new List<string> { "859-98-0987", "800-102-1100" };
+            foreach (PiiEntity entity in entities)
+            {
+                Assert.IsTrue(entitiesList.Contains(entity.Text));
+                Assert.Greater(entity.Length, 0);
+            }
+        }
+
+        [Test]
+        public async Task RecognizePiiEntitiesWithLanguageTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            string document = singleEnglish;
+
+            PiiEntityCollection entities = await client.RecognizePiiEntitiesAsync(document, "en");
+
+            Assert.AreEqual(2, entities.Count);
```

Comment:
Can you also include a test for redacted text? I think verifying on the actual string should be ok, since the redaction really shouldn't changeo n the service side (if it does, it might mean something is wrong with the service)

Reply:
Ah good point!. Will add